test/acceptance/static_page_test.rb

```diff
@@ -19,10 +19,10 @@ def test_about_exists
   def test_getting_started_exists
     Xapi.stub(:get, [200, File.read("./test/fixtures/tracks.json")]) do
       visit '/'
-      click_on 'Welcome'
+      #click_on 'Welcome'
     end
 
-    assert_content 'Exercises'
-    assert_content 'Nitpicking'
+    assert_content 'How it works'
+    assert_content 'Help'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.